### DISCUSSION TITLE
فصل های کتاب

### DISCUSSION THREAD

**Discussion by nsrmhjl** [nsrmhjl/ptDtSc#9] [Announcements]
سلام @AlirezaDoustmohammadi 
من تصمیم گرفتم که فصل های working with genomic data و population genetics رو درس ندم. کدها بسیار پیچیده هست و برای فهمیدن یک دونش یک روز وقت لازمه.

**Comment by AlirezaDostmohammadi** [comment]
سلام دکتر جان خوبید؟ پس فصل بعدی میشه فصل ۷  Phylogenetics درسته؟

**Comment by nsrmhjl** [comment]
بله
خیلی ممنون

**Comment by nsrmhjl** [comment]
ببخشید @AlirezaDoustmohammadi 
گفتی تو فیلوژنی چه مشکلی پیش اومد؟
من فقط explore  رو می بینم.

**Comment by AlirezaDostmohammadi** [comment]
سلام دکتر خوبید؟
فصل 7 رو دارم آپلود میکنم. فقط کد ها رو ران نکردم به جز کدهای فایلExploration.py

چون بقیه فایل ها تو گوگل کولب هر چی تست کردم رانش تموم نمیشه در مدت زمانه کم
